specutils/analysis/flux.py

```diff
@@ -116,3 +124,65 @@ def _compute_equivalent_width(spectrum, continuum=1, regions=None):
     ew =  dx - (line_flux / continuum)
 
     return ew.to(calc_spectrum.spectral_axis.unit)
+
+def is_continuum_near_zero(spectrum, eps=0.01):
+    """
+    Determine if the typical baseline of this spectrum is near zero. I.e., an
+    estimate of whether or not the continuum has been subtracted.
+
+    Parameters
+    ----------
+    spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
+        The spectrum object over which the width will be calculated.
+
+    eps: float
```

Comment:
`eps` -> `threshold`.  And also should say that it can be a flux-unit quantity or a float/dimensionless qunatity, and that determines if it's a flux threshold vs a "signal/noise" threshold

Reply:
already done